app/services/state_file/state_information_service.rb

```diff
@@ -0,0 +1,43 @@
+module StateFile
+  class StateInformationService
+    class << self
+      def active_states # TODO thinking this could also be called `active_state_codes` or just `state_codes`?
+        STATE_INFO.keys.map(&:to_s)
+      end
+
+      def state_name(state_code)
+        unless active_states.include?(state_code)
+          raise StandardError, state_code
+        end
+
+        States.name_for_key(state_code.upcase)
```

Comment:
Feeling less & less convinced that keeping the States class around is a good idea. I think I'd prefer that everyone come to this class for a display name, and that it be stored in the STATE_INFO hash.

Reply:
well, the States class is unfortunately used in a bunch of places so idk about getting rid of it but i do agree that it would make sense for this service to have its own source of truth about state names